lib/prepare_indices/create_indices.rb

```diff
@@ -10,7 +10,7 @@ def perform(params)
     private
 
       def start(params)
-        client = params[:es]
+        client = Elasticsearch::Client.new(host: params[:es], log: true)
```

Comment:
šlo by to logování vypnout přes parametr? Chci logovat když si něco pouštím na lokále, na serveru to naopak kýžené není.